content/applications/marketing/events/revenues_report.rst

```diff
@@ -0,0 +1,161 @@
+===============
+Revenues report
+===============
+
+The Odoo *Events* application provides the ability to create custom reports, based on event-related
+data and analytics. These reports can either be focused on the *Attendees* or the *Revenues*. Both
+reporting pages offer their own unique array of metrics and options.
+
+The following documentation focuses solely on the reporting options related to event *Revenues*.
+
+Revenues reporting page
+=======================
+
+To access the *Attendees* reporting page in the Odoo *Events* application, navigate to
+:menuselection:`Events app --> Reporting --> Revenues`.
+
+.. image:: revenues_report/default-view.png
+   :align: center
+   :alt: The default view of the Revenues reporting page in the Odoo Events application.
+
+By default, the :guilabel:`Revenues` reporting page appears as a graph (a :icon:`fa-line-chart`
+:guilabel:`(line chart)` with :icon:`fa-database` :guilabel:`(stacked)` data). The default filters,
+:guilabel:`Non-free tickets` and :guilabel:`Event Start Date: (current year)`, are present in the
+:guilabel:`Search...` bar.
+
+.. tip::
+   To learn more about the various graph views (and graph view options), refer to the documentation
+   on :ref:`Graph views <reporting/using-graph>`.
+
+The :guilabel:`Attendees` reporting page can also be viewed as a :ref:`pivot table
+<reporting/views/pivot>`, by clicking the :icon:`oi-view-pivot` :guilabel:`(pivot table)` icon in
+the upper-right corner.
+
+Measures
+--------
+
+Choosing specific :ref:`Measures <reporting/choosing-measures>` is a quick way to customize
+reporting pages.
+
+Regardless of view (graph or pivot table), the measures on the :guilabel:`Revenues` reporting page
+are as follows: :guilabel:`Revenues`, :guilabel:`Untaxed Revenues`, and :guilabel:`Count`.
+
+.. note::
+   In the default graph view of the :guilabel:`Revenues` reporting page, only the
+   :guilabel:`Revenues` option is set in the :guilabel:`Measures` drop-down menu.
+
+   When the pivot option is selected, via the :icon:`oi-view-pivot` :guilabel:`(pivot table)` icon
+   in the upper-right corner, all :guilabel:`Measures` options are selected, by default.
+
+.. tip::
+   In graph view, only one of the :guilabel:`Measures` can be selected at a time.
```

Comment:
I think these two blocks can be combined...
In the default graph view, only one of the measures can be selected at a time, the revenues option is selected by default, etc.
In the pivot view, etc...